src/OpenURLServer.js

```diff
@@ -41,6 +38,24 @@ class OpenURLServer {
       });
     }
 
+    const svcCfg = cfg.getValues()?.services?.[symbol] ?? cfg.getValues();
+    if (svcCfg?.reqIdHeader) {
+      let fromHeader = ctx.req.headers?.[svcCfg?.reqIdHeader];
```

Comment:
You don't need `?.` here, as you've already tested it.

Reply:
Apologies, bad habit from Groovy where it's needed at the top level (eg. it'd throw an exception if there was no `reqIdHeader` key).